File: Simplace/SimplaceReferenceETPM/src/cs/SimplaceReferenceETPM/ReferenceETPM_Exogenous.cs
using System;
using System.Collections.Generic;

public class ReferenceETPM_Exogenous 
{
    private double _iNetRadiation;
    private double _iActualVapourPressure;
    private double _iTMax;
    private double _iTMin;
    private double _iWindspeed;
    
    /// <summary>
    /// Constructor of the ReferenceETPM_Exogenous component")
    /// </summary>  
    public ReferenceETPM_Exogenous() { }
    
    
    public ReferenceETPM_Exogenous(ReferenceETPM_Exogenous toCopy, bool copyAll) // copy constructor 
    {
        if (copyAll)
        {
    
            iNetRadiation = toCopy.iNetRadiation;
            iActualVapourPressure = toCopy.iActualVapourPressure;
            iTMax = toCopy.iTMax;
            iTMin = toCopy.iTMin;
            iWindspeed = toCopy.iWindspeed;
        }
    }
    public double iNetRadiation
    {
        get { return this._iNetRadiation; }
        set { this._iNetRadiation= value; } 
    }
    public double iActualVapourPressure
    {
        get { return this._iActualVapourPressure; }
        set { this._iActualVapourPressure= value; } 
    }
    public double iTMax
    {
        get { return this._iTMax; }
        set { this._iTMax= value; } 
    }
    public double iTMin
    {
        get { return this._iTMin; }
        set { this._iTMin= value; } 
    }
    public double iWindspeed
    {
        get { return this._iWindspeed; }
        set { this._iWindspeed= value; } 
    }
}

File: Simplace/SimplaceReferenceETPM/src/java/SimplaceReferenceETPM/ReferenceETPM_Exogenous.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class ReferenceETPM_Exogenous
{
    private double iNetRadiation;
    private double iActualVapourPressure;
    private double iTMax;
    private double iTMin;
    private double iWindspeed;
    
    public ReferenceETPM_Exogenous() { }
    
    public ReferenceETPM_Exogenous(ReferenceETPM_Exogenous toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.iNetRadiation = toCopy.getiNetRadiation();
            this.iActualVapourPressure = toCopy.getiActualVapourPressure();
            this.iTMax = toCopy.getiTMax();
            this.iTMin = toCopy.getiTMin();
            this.iWindspeed = toCopy.getiWindspeed();
        }
    }
    public double getiNetRadiation()
    { return iNetRadiation; }

    public void setiNetRadiation(double _iNetRadiation)
    { this.iNetRadiation= _iNetRadiation; } 
    
    public double getiActualVapourPressure()
    { return iActualVapourPressure; }

    public void setiActualVapourPressure(double _iActualVapourPressure)
    { this.iActualVapourPressure= _iActualVapourPressure; } 
    
    public double getiTMax()
    { return iTMax; }

    public void setiTMax(double _iTMax)
    { this.iTMax= _iTMax; } 
    
    public double getiTMin()
    { return iTMin; }

    public void setiTMin(double _iTMin)
    { this.iTMin= _iTMin; } 
    
    public double getiWindspeed()
    { return iWindspeed; }

    public void setiWindspeed(double _iWindspeed)
    { this.iWindspeed= _iWindspeed; } 
    
}